__tests__/article-tests.js

```diff
@@ -154,4 +154,46 @@ describe.only('Patch article', () => {
     });
 });
 
+describe('Get articles by topic', () => {
+    it('Returns all articles when empty topic is passed', () => {
+        const topic = {topic: ''};
+        return request(app)
+            .get('/api/articles')
+            .send(topic)
+            .expect(200)
+            .then(({body}) => {
+                expect(body.articles).toHaveLength(5);
+            });
+    });
```

Comment:
We want topic to be a _query_, these are passed in on the url itself, e.g. /api/articles?topic=mitch

This is then accessible on the req.query object within your controller

Reply:
Ah ok, I was thinking of security and so not having it in the url as looking at the data I can see it is a name. But should I change it?